app/screens/home/channel_list/categories_list/categories/body/index.ts

```diff
@@ -107,11 +108,17 @@ const enhance = withObservables(['category', 'isTablet', 'locale'], ({category,
             }),
         );
 
+    const emptyDmIds = queryEmptyDirectAndGroupChannels(database).observeWithColumns(['last_post_at']).pipe(
```

Comment:
I wonder if this is  better handled by adding this logic to the query in `observeSortedChannels`. Something like...
```
channels.extend(..., Q.on(MY_CHANNELS, 'last_post_at', Q.gt(0))
```